File: Samples/dtquick.js
const mysql = require('mysql');

// Establish database connection
const con = mysql.createConnection({
    host: 'localhost',
    user: 'root',
    password: '',
    database: 'newDB',
});

con.connect((err) => {
    if (err) throw err;
    console.log('Connected!');
});

// Build the query
const query = req.query['query'];
const totalRecords = req.query['totalRecords'];

// Execute the query and get the total number of records
con.query(query, (err, rows) => {
    if (err) throw err;

    con.query(totalRecords, (err, result) => {
        if (err) throw err;

        const response = {
            data: rows,
            found: rows.length,
            total: result[0].totalRecords,
        };

        // Send JSON to the client-side
        res.send(response);
    });
});

File: Samples/dtquick.py
import mysql.connector
import json

# Establish database connection
db = mysql.connector.connect(
    host="localhost",
    user="root",
    password="",
    database="newDB"
)

# Build the query
query = request.args['query']
totalRecords = request.args['totalRecords']

# Execute the query and get the total number of records
cursor = db.cursor()
cursor.execute(query)
rows = cursor.fetchall()
cursor.execute(totalRecords)
total = cursor.fetchone()[0]

# Format the response as JSON and send it back to the client-side
response = {"data": rows, "found": len(rows), "total": total}
json_response = json.dumps(response, default=str)

print(json_response)